### ISSUE TITLE
Crash in utils.py in user_or_ip with Django 2.0

### ISSUE THREAD

**Issue by lmdsp** [jsocol/django-ratelimit#134]
utils.py l.28 uses `user.is_authenticated()` which is [deprecated in Django 1.10](https://docs.djangoproject.com/en/2.0/releases/1.10/#using-user-is-authenticated-and-user-is-anonymous-as-methods) and [removed in 2.0](https://docs.djangoproject.com/en/2.0/releases/2.0/#features-removed-in-2-0),
raising at TypeError exception ('bool' object is not callable).
The function has been replaced with the corresponding property `user.is_authenticated`

```python
def user_or_ip(request):
    if request.user.is_authenticated(): # Removed in Django 2.0
        return str(request.user.pk)
    return request.META['REMOTE_ADDR']
```

Since Django 2.0:

> Using User.is_authenticated() and User.is_anonymous() as methods rather than properties is no longer supported.

**Comment by adamchainz** [comment]
django-ratelimit is tested with django 2.0, is this line just not hit in the tests? And presumably you're using the latest version of django-ratelimit?

**Comment by jsocol** [comment]
Here's an equivalent [patch in waffle](https://github.com/jsocol/django-waffle/pull/259) if anyone wants to open a PR, but yeah it must not be exercised by the tests. Or was it changed after 2.0b1?

**Comment by adamchainz** [comment]
The tests don't pin 2.0b1, they allow 2.0rc1 and 2.0 to be used: https://github.com/jsocol/django-ratelimit/blob/master/tox.ini#L15 . As long as they've run since the release then we'd know..

**Comment by lmdsp** [comment]
@adamchainz This is django-ratelimit 1.1.0 of course. It's one of the functions to compute key values.

**Comment by jsocol** [comment]
Travis doesn't use tox, it [does pin 2.0b1](https://github.com/jsocol/django-ratelimit/blob/master/.travis.yml#L8), so the [last build](https://travis-ci.org/jsocol/django-ratelimit/builds/310558604) passed 2.0b1

**Comment by adamchainz** [comment]
Oh yeah 🤦‍♂️

**Comment by jsocol** [comment]
I think this was fixed by #136
